spatial_navigation.js

```diff
@@ -841,7 +841,7 @@
   }
 
   function onKeyDown(evt) {
-    if (!_sectionCount || _pause) {
+    if (!_sectionCount || _pause || evt.altKey || evt.ctrlKey || evt.metaKey || evt.shitKey) {
```

Comment:
This file adheres the 80-character limit. Could you break it into two lines? Thanks.